README.md

```diff
@@ -25,6 +25,7 @@ Plugin | Support Dockerfile | Support `cloudbuild.yaml` | Support LLB
 [OpenShift Image Builder](https://github.com/openshift/imagebuilder) | Planned | |
 [Orca](https://github.com/cyphar/orca-build) | Planned | |
 [Google Cloud Container Builder](https://cloud.google.com/container-builder/) | Planned | Planned |
+[kaniko](https://github.com/GoogleCloudPlatform/kaniko) | Yes | |
```

Comment:
As kaniko plugin does not exist atm, probably s/Yes/Planned/ might be accurate.
But I think "Yes" is ok as long as we soon implement the plugin. 😃

Reply:
haha sounds good!